src/features/operator-profile/ui/OperatorProfile/OperatorProfile.test.tsx

```
@@ -0,0 +1,170 @@
+  description: 'Experienced operator.',
+  philosophy: 'Customer first.',
+  photo: 'test-photo.jpg',
+  companyName: 'Tech Solutions',
+  createdAt: '2023-01-01T00:00:00Z',
+  updatedAt: '2023-01-01T00:00:00Z',
+  userId: 10,
+  website: 'https://techsolutions.com',
+  status: 'approved',
+  email: 'test@example.com',
+  phone: '+380501112233',
+};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)